include/dgl/runtime/shared_mem.h

```diff
@@ -42,6 +36,14 @@ class SharedMemory {
   size_t size;
 
  public:
+  /*
+   * \brief the name of the object.
+   *
+   * In Unix, shared memory is identified by a file. Thus, `name` is actually
+   * the file name that identifies the shared memory.
+   */
+  std::string name;
```

Comment:
You can define a getter function `std::string name() const` for private members.